pkg/controllers/cloud/configmap_controller.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2023 Antrea Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cloud
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
+	"gopkg.in/yaml.v2"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"antrea.io/nephe/pkg/controllers/config"
+)
+
+type ConfigMapReconciler struct {
+	client.Client
+	Log              logr.Logger
+	Scheme           *runtime.Scheme
+	ControllerConfig *config.ControllerConfig
+}
+
+func (r *ConfigMapReconciler) Reconcile(_ context.Context, request ctrl.Request) (ctrl.Result, error) {
+	_ = r.Log.WithValues("configmap", request.NamespacedName)
+        if request.Namespace != config.ConfigMapNamespace || request.Name != config.ConfigMap {
+		return ctrl.Result{}, nil
+	}
+	configMap := corev1.ConfigMap{}
+	if err := r.Get(context.TODO(), request.NamespacedName, &configMap); err != nil {
+		if !errors.IsNotFound(err) {
+			r.Log.V(0).Info("Error getting ConfigMap", "ConfigMap", request.NamespacedName)
```

Comment:
Log message not needed. 
Also the check `if !errors.IsNotFound(err) {` is not needed. If there is any error while in Get, you should return an error.

Reply:
Solved